requirements.txt

```diff
@@ -0,0 +1,2 @@
+le-utils==0.0.9rc23
```

Comment:
Good point. Technically, this line isn't required, but I think since this chef also uses `le-utils` directly (specifically, `le_utils.constants.licenses`), I'd prefer to keep an explicit dependency in this chef to `le-utils`, but perhaps changing `==` to a `>=`. Thanks for pointing this out though!